src/PimcoreDevkitBundle/Command/DeleteFolderCommand.php

```diff
@@ -24,100 +25,104 @@ class DeleteFolderCommand extends AbstractCommand
+        $helpMessage .= 'Example usage:'.PHP_EOL;
+        $helpMessage .= 'bin/console devkit:deletefolder -t object -f /bigUglyFolder';
 
         $this
-            ->setName("devkit:deletefolder")
-            ->setDescription("Deletes folder from given tree")
+            ->setName('devkit:deletefolder')
+            ->setDescription('Deletes folder from given tree')
             ->setHelp($helpMessage)
-            ->addOption("type", "t", InputOption::VALUE_REQUIRED, "Tree type (object, document, asset)")
-            ->addOption("folder", "f", InputOption::VALUE_REQUIRED, "Folder path");
+            ->addOption('type', 't', InputOption::VALUE_REQUIRED, 'Tree type (object, document, asset)')
+            ->addOption('folder', 'f', InputOption::VALUE_REQUIRED, 'Folder path');
     }
 
     /**
      * {@inheritdoc}
      */
-    protected function execute(InputInterface $input, OutputInterface $output) : void
+    protected function execute(InputInterface $input, OutputInterface $output): void
     {
-        $type = $input->getOption("type");
-        if(!$type) {
-            $output->writeln("Missing -t option.");
+        $type = $input->getOption('type');
+        if (!$type) {
+            $output->writeln('Missing -t option.');
+
             return;
         }
 
-        $path = $input->getOption("folder");
-        if(!$path) {
-            $output->writeln("Missing -f option.");
+        $path = $input->getOption('folder');
+        if (!$path) {
+            $output->writeln('Missing -f option.');
+
             return;
         }
 
         try {
             switch ($type) {
-                case "object":
+                case 'object':
                     $this->deleteObject($path);
                     break;
-                case "document":
+                case 'document':
                     $this->deleteDocument($path);
                     break;
-                case "asset":
+                case 'asset':
                     $this->deleteAssetFolder($path);
                     break;
                 default:
-                    $output->writeln("Incorrect tree type, allowed types: object, document, asset.");
+                    $output->writeln('Incorrect tree type, allowed types: object, document, asset.');
+
                     return;
             }
 
-            $output->writeln("Folder ".$path." has been deleted.");
+            $output->writeln('Folder '.$path.' has been deleted.');
         } catch (\Exception $e) {
             $output->writeln($e->getMessage());
         }
-
     }
 
     /**
      * @param string $path
+     *
      * @throws \Exception
      */
     protected function deleteObject(string $path)
     {
         $folder = Folder::getByPath($path);
-        if(!$folder) {
-            throw new \Exception("Object with path ".$path." does not exist.");
+        if (!$folder) {
+            throw new \Exception('Object with path '.$path.' does not exist.');
```

Comment:
maybe it is better to use subsidiarity here? not obligatory, but ihmo more readable.
Same with orher concatenations here.